commit ba02abf58f8d0405140e658d8238b69e89120576
Author: Jason Gerecke <jason.gerecke@wacom.com>
Date:   Wed Jan 6 13:01:29 2021 -0800

    data: Add ISDv4 51e9 (Lenovo ThinkPad P15 Gen 1) (#325)
    
    Tablet definition was tested and verified to work by the bug reporter.
    
    Signed-off-by: Jason Gerecke <jason.gerecke@wacom.com>

diff --git a/data/isdv4-51e9.tablet b/data/isdv4-51e9.tablet
new file mode 100644
index 0000000..8684b26
--- /dev/null
+++ b/data/isdv4-51e9.tablet
@@ -0,0 +1,22 @@
+# Lenovo ThinkPad P15 Gen 1
+# Sensor Type: AES
+# Features: Touch (Integrated), Tilt
+# HW Resolution: 34422 x 19362 (2540 x 2540 lpi)
+#
+# Autogenerated from sysinfo.VSiBXWPx2J
+# From private bug report
+
+[Device]
+Name=ISDv4 51e9
+ModelName=
+DeviceMatch=usb:056a:51e9
+Class=ISDV4
+Width=14
+Height=8
+IntegratedIn=Display;System
+Styli=@isdv4-aes;
+
+[Features]
+Stylus=true
+Touch=true
+Buttons=0
